### PR TITLE
More on re-brand fixes and improvements

### LINKED ISSUE #497: api-doc page reporting wrong account sometimes
The page is reporting wrong account if a request uses secretPhrase that has the char `#` in it.

Example: login with passphrase `1SceaPIT#ZPge4fsrCS4Cvu3` and your account is **BURST-NH8N-8ET3-APBM-9M2MV**
Use api-doc page and try to `getAccountId` with same password, it will show account **BURST-JLSG-8BJH-4QVA-C9GT2**

Problem: The api-doc page is using jquery ajax function in a manner that the data will be parsed by jquery, leading password to be cropped on char `#`.

Solution: Send data to ajax function as an object.

Code:

```diff
diff --git a/src/brs/http/APITestServlet.java b/src/brs/http/APITestServlet.java
index a76943f9..2fce16d0 100644
--- a/src/brs/http/APITestServlet.java
+++ b/src/brs/http/APITestServlet.java
@@ -49,14 +49,16 @@ public class APITestServlet extends HttpServlet {
       + "        }\n"
       + "        function submitForm(form) {\n"
       + "            var url = '/burst';\n"
+      + "            var data = {};\n"
       + "            for (i = 0; i < form.elements.length; i++) {\n"
       + "                if (form.elements[i].type != 'button' && form.elements[i].value && form.elements[i].value != 'submit') {\n"
-      + "                    url += ((i == 0 && '?') || '&') + form.elements[i].name + '=' + form.elements[i].value;\n"
+      + "                    data[form.elements[i].name] = form.elements[i].value;\n"
       + "                }\n"
       + "            }\n"
       + "            $.ajax({\n"
       + "                url: url,\n"
       + "                type: 'POST',\n"
+      + "                data: data, \n"
       + "            })\n"
       + "            .done(function(result) {\n"
       + "                var resultStr = JSON.stringify(JSON.parse(result), null, 4);\n"
```

### PR DESCRIPTION
**Pull request by jjos2372**
- Fix some issues with the new address prefix #503 #504 
 - api-doc problem, PR #498 that fixes #497 has a side effect that the "Open GET URL" stopped working.

This should also fix the GET URL think and hopefully also fix the original issue, please check if that is the case.

unknown (1)

### CODE REVIEW

File: src/brs/http/APITestServlet.java [hunk 1]
```diff
@@ -24,7 +24,7 @@
       + "    <meta charset=\"UTF-8\"/>\n"
       + "    <meta http-equiv=\"X-UA-Compatible\" content=\"IE=edge\">"
       + "    <meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">"
-      + "    <title>Burst http API</title>\n"
+      + "    <title>Signum http API</title>\n"
```
Comment (Zoh63392187) [inline src/brs/http/APITestServlet.java:27]: Maybe remove http as it could be https?

File: src/brs/http/APITestServlet.java [hunk 2]
```diff
@@ -79,7 +80,7 @@
       + "<div class=\"navbar navbar-default\" role=\"navigation\">"
       + "   <div class=\"container\" style=\"min-width: 90%;\">"
       + "       <div class=\"navbar-header\">"
-      + "           <a class=\"navbar-brand\" href=\"" + API.API_TEST_PATH + "\">Burst http API</a>"
+      + "           <a class=\"navbar-brand\" href=\"" + API.API_TEST_PATH + "\">Signum http API</a>"
```
Comment (Zoh63392187) [inline src/brs/http/APITestServlet.java:83]: Maybe remove http as it could be https?

**Review by Zoh63392187 — APPROVED** [review]
Just minor wording suggestion.

File: src/brs/http/APITestServlet.java (hunk 1 above)
Comment (jjos2372) [reply src/brs/http/APITestServlet.java]: good idea, http was removed.

File: src/brs/http/APITestServlet.java [hunk 3]
```diff
@@ -53,6 +53,7 @@
       + "            for (i = 0; i < form.elements.length; i++) {\n"
       + "                if (form.elements[i].type != 'button' && form.elements[i].value && form.elements[i].value != 'submit') {\n"
       + "                    data[form.elements[i].name] = form.elements[i].value;\n"
+      + "                    url += ((i == 0 && '?') || '&') + form.elements[i].name + '=' + form.elements[i].value;\n"
```
[the remark this answers is not in the document]
Comment (jjos2372) [reply src/brs/http/APITestServlet.java]: Thanks please check if bbd1fde fixes it.

**Review by deleterium — APPROVED** [review]
Nice!

**Pull request merged**